crates/iota-storage/src/http_key_value_store.rs

```diff
@@ -282,6 +282,42 @@ enum Value {
     TxToCheckpoint(CheckpointSequenceNumber),
 }
 
```

Comment:
Just a reminder the function `key_to_path_elements` was removed at https://github.com/iotaledger/iota/pull/5296

Reply:
Do we need to adapt the new`to_path_elements` then?

Reply:
key_to_path_elements was only made public upstream so we already have the equivalent if I'm not mistaken ?